app/backend/aquifers/models.py

```diff
@@ -0,0 +1,339 @@
+        primary_key=True, editable=False, verbose_name="Aquifer ID Number")
+    aquifer_name = models.CharField(max_length=100)
+    location_description = models.CharField(
+        max_length=100, blank=True, verbose_name='Description of Location')
+    material = models.ForeignKey(
+        AquiferMaterial,
+        db_column='aquifer_material_code',
+        blank=True,
+        null=True,
+        on_delete=models.PROTECT,
+        verbose_name="Material Reference",
+        related_name='aquifers')
+    subtype = models.ForeignKey(
+        AquiferSubtype,
+        db_column='aquifer_subtype_code',
+        blank=True,
+        null=True,        
+        on_delete=models.PROTECT,
+        verbose_name="Subtype Reference",
+        related_name='aquifers')
+    area = models.DecimalField(
+        max_digits=5, decimal_places=1, blank=True, null=True, verbose_name='Size (square km)')
+    productivity = models.ForeignKey(
+        AquiferProductivity,
+        db_column='aquifer_productivity_code',
+        blank=True,
+        null=True,        
+        on_delete=models.PROTECT,
+        verbose_name="Productivity Reference",
+        related_name='aquifers')
+    vulnerability = models.ForeignKey(
+        AquiferVulnerability,
+        db_column='aquifer_vulnerability_code',
+        blank=True,
+        null=True,        
+        on_delete=models.PROTECT,
+        verbose_name="Vulnerability Reference",
+        related_name='aquifers')
+    demand = models.ForeignKey(
+        AquiferDemand,
+        db_column='aquifer_demand_code',
+        blank=True,
+        null=True,
+        on_delete=models.PROTECT,
+        verbose_name="Demand Reference",
+        related_name='aquifers')
+    known_water_use = models.ForeignKey(
+        WaterUse,
+        db_column='water_use_code',
+        blank=True,
+        null=True,        
+        on_delete=models.PROTECT,
+        verbose_name="Known Water Use Reference",
+        related_name='aquifers')
+    quality_concert = models.ForeignKey(
+        QualityConcern,
+        db_column='quality_concern_code',
+        blank=True,
+        null=True,        
+        on_delete=models.PROTECT,
+        verbose_name="Quality Concern Reference",
+        related_name='aquifers')
+    litho_stratographic_unit = models.CharField(
+        max_length=100, blank=True, verbose_name='Lithographic Stratographic Unit')
+    mapping_year = models.PositiveIntegerField(
+        validators=[
+            MinValueValidator(1990), 
+            MaxValueValidator(timezone.now().year)],
+        blank=True,
+        null=True,                
+        verbose_name="Date of Mapping",    
+        help_text="Use the following format: <YYYY>")
+
+    notes = models.TextField(
+        max_length=2000,
+        blank=True,
+        null=True,
+        verbose_name='Notes on Aquifer, for internal use only.')
+
+
+
+    """
+ aquifer_subtype_code        | character varying(10)          | not null  |
```

Comment:
Please insert tabs to align comments with code

Reply:
this placeholder comment no longer needed so dropped completely.